update badges

benlorenz opened the pull request:
they were still using travis, add dev docs

kalmarek reviewed (commented):
should the `version` badge be graphics, or a link to the release?

benlorenz commented:
> should the `version` badge be graphics, or a link to the release?

I removed the link because it was just pointing to travis. But now I added a link that should always point to the _latest_ release.
